test/powershell/Language/Parser/RedirectionOperator.Tests.ps1

```diff
@@ -47,6 +47,14 @@ Describe "Redirection operator now supports encoding changes" -Tags "CI" {
     $availableEncodings = (get-command out-file).Parameters["Encoding"].Attributes.ValidValues
 
     foreach($encoding in $availableEncodings) {
+        if ($encoding -eq "default") {
+            # [System.Text.Encoding]::Default is exposed by 'System.Private.CoreLib.dll' at
+            # runtime via reflection. However,it isn't exposed in the reference contract of
+            # 'System.Text.Encoding', and therefore we cannot use 'Encoding.Default' in our
+            # code. So we need to skip this encoding in the test.
+            continue

```

Comment:
Can you set a variable that controls whether the It in this loop is skipped? It'll satisfy the tracked skipping requirement that way.